tests/02_eradiate/01_unit/scenes/atmosphere/test_molecular_atmosphere.py

```diff
@@ -39,27 +39,20 @@ def test_molecular_atmosphere_afgl_1986(mode_ckd):
         mi_params.update(params.render(ctx))
 
 
-@pytest.fixture
-def ussa76_approx_test_absorption_data_set():
-    return data_store.fetch("tests/spectra/absorption/us76_u86_4-spectra-4000_25711.nc")
-
-
-def test_molecular_atmosphere_ussa_1976(
-    mode_mono, ussa76_approx_test_absorption_data_set
-):
+@pytest.mark.slow
```

Comment:
Is this test slower than before?

Reply:
I don't remember: is this data shipped as part of the data submodule, or is it hosted on the FTP?

Reply:
It's here: https://eradiate.eu/data/store/unstable/spectra/absorption/us76_u86_4/

Reply:
I guess this is why we had a test dataset in the first place. Downloads for testing are usually not great, in particular because tests may fail just because data is missing. We should have a fixture which declares required data and may skip tests if data is inaccessible. I'll create an issue for that.

Reply:
There is `skipif_data_not_found` in `test_tools/util.py`; I could use that perhaps?

Reply:
Right, I didn't remember it. Indeed you can.